src/App/Fossa/Analyze/Types.hs

```diff
@@ -113,9 +113,20 @@ data DiscoveredProjectIdentifier = DiscoveredProjectIdentifier
   }
   deriving (Eq, Ord, Show)
 
+{-# WARNING analyzeProject' "analyzeProject' has been renamed to analyzeProjectStaticOnly. Please use that name for all future implementations." #-}
 class AnalyzeProject a where
-  -- | Analyze a project with any tactic.
+  -- | Analyze a project with any tactic including dynamic ones.
   analyzeProject :: AnalyzeTaskEffs sig m => FoundTargets -> a -> m DependencyResults
 
   -- | Analyze a project with only static tactics.
-  analyzeProject' :: AnalyzeStaticTaskEffs sig m => FoundTargets -> a -> m DependencyResults
+  -- Static here generally means not using any external utilities.
+  -- An exception is berkeleydb which calls a program embedded in the CLI.
+  analyzeProjectStaticOnly :: AnalyzeStaticTaskEffs sig m => FoundTargets -> a -> m DependencyResults
+  -- This is required to satisfy the MINIMAL pragma below and for compatibility with existing consumers of AnalyzeProject.
+  -- Please don't use this default implementation for any new code.
+  analyzeProjectStaticOnly = analyzeProject'
```

Comment:
Why not just refactor callsites and drop this?

Reply:
wait it looks like we did just refactor the callsites- why even have `analyzeProject'` still?

Reply:
You're right. I was initially thinking that maybe `hubble` was using that name and didn't want to force them to do more work. I will drop it.

Reply:
I've deleted `analyzeProject'`.